src/WinUIEx/Placement.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WinUIEx
+{
+    /// <summary>
+    /// The edge positions MoveToPosition can move to
+    /// </summary>
+    public enum Placement
+    {
+        TopLeft,
+        TopCenter,
+        TopRight,
+        MiddleLeft,
+        MiddleCenter,
+        MiddleRight,
+        BottomLeft,
+        BottomCenter,
+        BottomRight
```

Comment:
Each value needs a `<summary/>` (you should be seeing build warnings for this)

Reply:
Comments would make it harder to read, but I've added them to increase consistency with the rest of the solution.